configuration.json

```diff
@@ -6,6 +6,6 @@
 
                 {"name":"watch", "settings":  {"timeZone":"Europe/Warsaw", "calendar": "http://localhost:8000/calendar.json"} },
                 {"name":"watch", "settings":  {"timeZone":"Europe/Warsaw", "calendar": "http://localhost:8000/calendar.json"} },
-                {"name":"watch", "settings":  {"timeZone":"Europe/Warsaw", "calendar": "http://localhost:8000/calendar.json"} }
+                {"name": "harvest", "settings": {"accessToken": "", "refreshToken": "", "expirationTime": 0, "refreshInterval": 3600}}
```

Comment:
Since it refreshes not only the token, but also the view, I think the simpler name is more suitable.